tests/phpunit/tests/functions/wpGetRawReferer.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * Tests for the wp_referer_field() function.
+ *
+ * @since 6.0.0
+ *
+ * @group functions.php
+ * @covers ::wp_get_raw_referer
+ */
+class Tests_Functions_wpGetRawReferer extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 55729
+	 */
+	public function test_wp_get_original__wp_http_referer_referer() {
+
```

Comment:
```suggestion
```

:1234: Applies throughout, no line break at the start of methods per WP coding standards.